### PR TITLE
Updated Models, Updated Auth, Updated JWT, Added JWT Secret, Cleanup,…

### PR DESCRIPTION
**Pull request by petersteele111**
… and spell check

### CODE REVIEW

File: backend/app/schemas.py [hunk 1]
```diff
@@ -14,22 +14,24 @@ class UserCreate(BaseModel):
...
+    user_is_medical_professional: bool
+    user_is_verified: bool
+    user_is_volunteer: bool
 
     class Config:
         orm_mode = True
 
 
 class Login(BaseModel):
-    username: str
```
Comment (Celeo) [inline backend/app/schemas.py]: This class isn't actually used anywhere, so it can just be removed.

File: backend/app/main.py [hunk 2]
```diff
@@ -41,11 +41,12 @@
     Returns:
         The generated OAuth token information
     """
-    user = crud.check_user(db, form_data.username, form_data.password)
+    # form_data.username will need to be the email the user signed up with. user_email is a str
+    user = crud.check_user(db, form_data.username, form_data.password)  # Shadows name user from outer scope
```
Comment (Celeo) [inline backend/app/main.py]: I don't understand the comment? 😕

File: backend/app/JWT.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+def get_jwt_key():
```
Comment (Celeo) [inline backend/app/JWT.py]: This is slightly better than "TEST_VALUE_PLEASE_CHANGE" but we still cannot have the key in code at all - it'll have to loaded from a secure environment.

Comment (Celeo) [reply backend/app/JWT.py]: For example, if we are going to be using AWS, then we'd want the QA and PROD _environments_ to have AWS Creds to pull the key from AWS SecretsManager. Dev, QA, and PROD would all have different keys.

File: backend/app/schemas.py (hunk 1 above)
Comment (petersteele111) [reply backend/app/schemas.py]: I can remove it

File: backend/app/main.py (hunk 2 above)
Comment (petersteele111) [reply backend/app/main.py]: It was like 4am I wasn't thinking straight lmao

Comment (Celeo) [reply backend/app/main.py]: Comment is still there?

**Pull request merged**